README.md

```diff
@@ -256,6 +256,69 @@ To check whether dnsmasq name server at 127.0.0.1 can resolve subdomains of dev.
 app.dev.gov.uk.		0	IN	A	127.0.0.1
 ```
 
+### How to: replicate data locally
+
+There may be times when a full database is required locally.  The following sections give examples of how to replicate this data from integration.  All examples reqire pv, which can be installed on a Mac using Brew (`brew install pv`).
+
+#### MySQL
+
+1. Download the relevant database dump from the [AWS S3 Bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups/mysql/?region=eu-west-1&tab=overview)
+
+2. Import the file into the local MySQL database, e.g. for Whitehall:
+
+```
+govuk-docker compose up -d mysql
+pv whitehall_production.dump.gz | gunzip | govuk-docker compose run mysql mysql -u root --password=root whitehall_development
+```
+
+#### PostgreSQL
+
+1. Download the relevant database dump from the [AWS S3 Bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups/postgres/?region=eu-west-1&tab=overview)
+
+2. Drop and recreate any existing database, e.g. for Publishing API:
+
+```
+govuk-docker compose up -d postgres
+govuk-docker compose run postgres /usr/bin/psql -U postgres -qAt DROP DATABASE IF EXISTS "publishing-api"
+govuk-docker compose run postgres /usr/bin/createdb -U postgres publishing-api
```

Comment:
Might be simpler to do this with rails - though not sure we need it if we're not doing this for any other dbs?

Reply:
I've added a similar step for MySQL.  Mongo also has the `--drop` flag, which replicates the behaviour of dropping then recreating the database.

Maybe when this is refactored to be slightly more automated, we can look into a way of doing this with Rails?